sys/arm/arm/pmap-v6.c

```diff
@@ -1090,6 +1090,8 @@ pmap_kextract(vm_offset_t va)
 	pt1_entry_t pte1;
```

Comment:
i should probably copy the comment above this function definition to all implementations.